src/components/BaseCanvas/BaseCanvas.component.tsx

```diff
@@ -22,16 +23,17 @@ export const BaseCanvas: FC<BaseCanvasProps> = ({
   fov = 50,
   position = new Vector3(0, 0, 5),
   style,
-  dpr = [BASE_DPR * 0.5, 2],
-  className
+  dpr = [BASE_DPR * 0.5, BASE_DPR * 0.75],
```

Comment:
Testing with reduced dpr values, probably wont be final

Reply:
Might set the top range to 1, then we can handover implementation details on the best value(s)